tol_lab_share/constants.py

```diff
@@ -28,6 +28,9 @@
 INPUT_CREATE_LABWARE_MESSAGE_SAMPLE_STUDY_UUID = "studyUuid"
 INPUT_CREATE_LABWARE_MESSAGE_SAMPLE_COLLECTION_DATE = "sampleCollectionDateUtc"
 INPUT_CREATE_LABWARE_MESSAGE_SAMPLE_VOLUME = "volume"
+INPUT_CREATE_LABWARE_MESSAGE_SAMPLE_ACCESSION_NUMBER = "accessionNumber"
```

Comment:
Advisory: INPUT_CREATE_LABWARE_MESSAGE is listed quite a few times. This would indicate to me that it should be ab object or a class.

Reply:
I could create group of constants with classes, I'll have a look at it to change it